tests/unittests/utils/test_depth_anything.py

```
@@ -228,3 +245,382 @@ def test_none_frame_size_skips_resize(self):
+    """The config reaches self._model.inference, not just the config object."""
+
+    @staticmethod
+    def _make_model(config_dict):
+        from fiftyone.utils.depth_anything import (
+            DepthAnythingV3Model,
+            DepthAnythingV3ModelConfig,
+        )
+
+        calls = []
+
+        def _fake_inference(inputs, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fake_inference`

(ANN202)